libraries/AP_ADSB/AP_ADSB.h

```diff
@@ -293,6 +301,32 @@ class AP_ADSB {
         ALL             = 2
     };
 
+    // #if HAL_ADSB_SAGETECH_MXS_ENABLED
+    struct {
```

Comment:
I think we got some trick on AP_Rangefinder to add some specific parameter to a sole backend. But that won't be blocking

Reply:
all of this needs to get moved up into the mxs driver